resources/assets/js/components/JobBuilder/Review/JobReview.tsx

```diff
@@ -202,10 +202,10 @@ const JobReviewSection: React.FunctionComponent<JobReviewSectionProps> = ({
             data-c-grid-item="tp(1of2)"
             data-c-alignment="base(centre) tp(right)"
           >
-            <a href={link}>
+            <Link href={link} title={linkLabel}>
               <i data-c-colour="c2" className="fas fa-edit" />
```

Comment:
@tristan-orourke just wanted to point out my change here, fragments are only required if you're returning sibling tags from a component. You can pass any amount of stuff (even as siblings) as children, and as long as the consuming component only returns a single tag it's fine.

Reply:
Awesome, thanks.